python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -1008,6 +1026,45 @@ def resolve_autoMaterializeAssetEvaluationsOrError(
             graphene_info=graphene_info, graphene_asset_key=assetKey, cursor=cursor, limit=limit
         )
 
+    def resolve_autoMaterializeTicks(
+        self, graphene_info, dayRange=None, dayOffset=None, limit=None, cursor=None, statuses=None
+    ):
+        from dagster._daemon.asset_daemon import (
+            FIXED_AUTO_MATERIALIZATION_ORIGIN_ID,
+            FIXED_AUTO_MATERIALIZATION_SELECTOR_ID,
+        )
+
+        before = None
+        if dayOffset:
+            before = pendulum.now("UTC").subtract(days=dayOffset).timestamp()
+        elif cursor:
+            parts = cursor.split(":")
```

Comment:
maybe this cursor logic should actually be centralized into some util class now that this logic is getting split into a couple different files.